doc/source/basics.rst

```diff
@@ -174,6 +176,18 @@ conventionally recorded in an environment variable.
     import os
     hosts = os.environ['EPICS_CA_ADDR_LIST']  # example: '172.17.255.255'
 
+Something simple like this would work but would only support one IP address in
```

Comment:
I'm fine with deleting the overly simple example above if you think it's not helpful.